app/components/content/mailing_list_component.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="action-container">
+  <div class="heading-container">
+    <h2 class="heading-m heading--box-purple">
+      Get the latest teaching information and support
+    </h2>
+  </div>
+
+  <div class="content">
+    <div class="inset">
+      <p>
+        Answer a couple of questions about yourself and we'll send you more information about teaching
+        salaries and benefits, as well as helpful tips and advice to get you one step closer to a career in teaching.
+      </p>
+
+      <%= form_with builder: GOVUKDesignSystemFormBuilder::FormBuilder, url: mailing_list_step_path(:name), scope: :mailing_list_steps_name, method: :put do |f| %>
+        <%= f.govuk_text_field :first_name, width: 'two-thirds', autocomplete: "given-name" %>
+        <%= f.govuk_text_field :last_name, width: 'two-thirds', autocomplete: "family-name" %>
+        <%= f.govuk_email_field :email, width: 'two-thirds', autocomplete: "email" %>
+        <%= f.hidden_field :accepted_policy_id, value: privacy_policy.id %>
+
+        <p>Your details are protected under the terms of our <%= link_to("privacy notice", privacy_policy_path(id: privacy_policy.id), { target: :blank }) %>.</p>
```

Comment:
@philspray We need to add in another line here to be consistent with our other sign-up pages:

**Our privacy notice explains how we use your personal data. It's important you read this before signing up to receive emails.**

![image](https://user-images.githubusercontent.com/87642394/217865829-0aa6f668-2ec2-43b9-b77f-a23305bbd58c.png)